pkg/option/config.go

```diff
@@ -2892,8 +2896,8 @@ func (c *DaemonConfig) Populate() {
 	c.MonitorAggregation = viper.GetString(MonitorAggregationName)
 	c.MonitorAggregationInterval = viper.GetDuration(MonitorAggregationInterval)
 	c.MonitorQueueSize = viper.GetInt(MonitorQueueSizeName)
-	c.MTU = viper.GetInt(MTUName)
```

Comment:
Was this change in the original PR? Please double-check that the backported version matches the original.